opencti-platform/opencti-front/tests_e2e/model/report.pageModel.ts

```diff
@@ -1,8 +1,13 @@
 import { Page } from '@playwright/test';
 
 export default class ReportPage {
+  pageUrl = '/dashboard/analyses/reports';
   constructor(private page: Page) {}
 
+  async goto() {
+    await this.page.goto(this.pageUrl);
```

Comment:
As far as possible, we should avoid using the goto function as it reloads the page (if I'm right) and slow the test, instead we should use the left navigation menu

Reply:
As far as I see it's often use for starting tests, I just replaced existing calls. It's true that it's taking time. i prefer having the page URL at only one place that is why I moved it.